Packs/Hudsonrock/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Hudsonrock",
+    "description": "Indicator enrichment from Hudsonrock free APIs",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Harri Ruuttila",
+    "url": "https://live.paloaltonetworks.com/t5/cortex-xsoar-discussions/bd-p/Cortex_XSOAR_Discussions",
```

Comment:
is this url intended? it points to general discussions

Reply:
I did not put it there, I will remove it. It has come from the demisto-sdk init